util/getMusic.py

```diff
@@ -0,0 +1,102 @@
+#Dry Popcorn -- Jason Lin, Jiajie Mai, Raymond Wu, Jerry Ye
+#SoftDev1 pd07
+#P01 -- ArRESTed Development
+#2018-11-26
+
+from urllib import request #stdlib
+import json #stdlib
+
+# util file for sending last.fm API requests, returning relevant information
+
+MUSIC_API_KEY = '7af285e176cbccfeb8a1b249c84479a1'
```

Comment:
I think one of the QAF posts said something about having a file that the user edits to paste their API keys into